rahtikirja.php

```diff
@@ -689,7 +688,20 @@ function lisaarivi($otunnus, $tuoteno, $kpl, $hinta = "") {
             if (mysql_num_rows($pakres) == 1) {
               $pakrow = mysql_fetch_assoc($pakres);
 
-              lisaarivi($otsikkonro, $pakrow["pakkausveloitus_tuotenumero"], $kollit[$i]);
+              $lasku_query = "SELECT rahtivapaa
```

Comment:
@tiere Rivillä 585 haetaan jo tietoja laskulta. Tuntuu turhalta/pahalta tehdä monta selectiä peräkkäin samasta taulusta. Voisko tässä käyttää jo olemaasaolevaa resulttia?